server/routes/api/parents/parentsDBHandler.ts

```diff
@@ -27,7 +27,7 @@ export default class DBHandler {
     parentId: number,
     babysitterId: number
   ): Promise<number | null> {
-    const query = `SELECT b.babysitter_id AS babysitter_id,
+    const query = `SELECT b.babysitter_id AS "babysitterId",
```

Comment:
it doesnt help that you change it here - you also need to change the parentId and change all the places that use this data